File: Java/src/classZoo/Bear.java
package classZoo;

public class Bear extends Animal {
	public Bear(String inputName, int inputID, float inputWeight, float inputHeight) {
		super(inputName, inputID, inputWeight, inputHeight);
	}
	
	public void printSpecies() {
		System.out.println("Animal Species: Bear");
	}
	
	public void animalSound() {
		System.out.printf("%s growls\n", this.getName());
	}
}

File: CPlusPlus/src/Bear.cpp
#include <iostream>
#include <string>
#include "../include/Bear.h"

namespace ClassZoo {
ClassZoo::Bear::Bear(std::string inputName, int inputID,
    float inputWeight, float inputHeight)
    : ClassZoo::Animal(inputName, inputID, inputWeight, inputHeight) {}
void ClassZoo::Bear::PrintSpecies() {
    std::cout << "Animal Species: Bear\n";
}
void ClassZoo::Bear::AnimalSound() {
    std::cout << ClassZoo::Animal::GetName() << " growls\n";
}
}